wirecardpaymentgateway/helper/OrderManager.php

```diff
@@ -23,10 +23,23 @@
  */
 class OrderManager
 {
+    use TranslationHelper;
+
+    /**
+     * @var string
+     * @since 2.8.0
+     */
+    const TRANSLATION_FILE = "ordermanager";
+
     const WIRECARD_OS_STARTING = 'WIRECARD_OS_STARTING';
     const WIRECARD_OS_AWAITING = 'WIRECARD_OS_AWAITING';
     const WIRECARD_OS_AUTHORIZATION = 'WIRECARD_OS_AUTHORIZATION';
 
+    const WIRECARD_OS_KEY_STARTED = 'order_state_payment_started';
```

Comment:
```suggestion
    const WIRECARD_OS_TRANSLATION_KEY_STARTED = 'order_state_payment_started';
```
I think its make sense define it so, cause **WIRECARD_OS_KEY_*** could be interpreted as key of object.